tests/Fixer/Import/FullyQualifiedStrictTypesFixerTest.php

```diff
@@ -1090,6 +1090,54 @@ public function getValue()
             null,
             ['import_symbols' => true],
         ];
+
+        // TODO: Ensure shortening for imported functions and constants
```

Comment:
Is shortening functions and constants not currently supported at all by the rule?

Reply:
Yes, the rule currently can shorten only class-like symbols. See #7460.